### PR TITLE
Clarify host parameter is optional in the README.

### PR DESCRIPTION
**Pull request by kurtisvg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ The full JDBC url should look like this:
 jdbc:mysql://google/<DATABASE_NAME>?cloudSqlInstance=<INSTANCE_CONNECTION_NAME>&socketFactory=com.google.cloud.sql.mysql.SocketFactory&useSSL=false&user=<MYSQL_USER_NAME>&password=<MYSQL_USER_PASSWORD>
```
Comment (saturnism) [inline README.md]: can we remove the `google` host here too?

File: README.md [hunk 2]
```diff
@@ -96,6 +96,8 @@ The full JDBC url should look like this:
...
 ```
 
+Note: The host portion of the JDBC url is currently unused, and has no effect on the connection process. The SocketFactory will get your instances IP address base on the provided `cloudSqlInstance` arg.
```
Comment (saturnism) [inline README.md]: url -> URL

Comment (saturnism) [reply README.md]: also, is this comment specific to MySQL? What about PostgresSQL?

File: README.md [hunk 3]
```diff
@@ -96,6 +96,8 @@ The full JDBC url should look like this:
 jdbc:mysql://google/<DATABASE_NAME>?cloudSqlInstance=<INSTANCE_CONNECTION_NAME>&socketFactory=com.google.cloud.sql.mysql.SocketFactory&useSSL=false&user=<MYSQL_USER_NAME>&password=<MYSQL_USER_PASSWORD>
 ```
 
+Note: The host portion of the JDBC url is currently unused, and has no effect on the connection process. The SocketFactory will get your instances IP address base on the provided `cloudSqlInstance` arg. 
+
 ##### Postgres
 
 Base JDBC url: `jdbc:postgresql://google/<DATABASE_NAME>`
```
Comment (saturnism) [inline README.md]: this should also be consistent

File: README.md (hunk 1 above)
Comment (kurtisvg) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (kurtisvg) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (kurtisvg) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (saturnism) [reply README.md]: while we are at it, can we just upper case the url? :D

Comment (kurtisvg) [reply README.md]: @saturnism - I don't see any that are still lowercase - do you see one in the current version? I was having some issues with Chrome/Github's search function stepping on each other, but I in the page preview they are all uppercase'd.

Comment (saturnism) [reply README.md]: i see them now. thnx!

**Pull request merged**